tests/integration/test_replicated_mutations/test.py

```diff
@@ -90,7 +90,7 @@ def do_insert(self, thread_num, partitions_num):
                 i += 1
 
             try:
-                print('thread {}: insert for {}: {}'.format(thread_num, date_str, ','.join(str(x) for x in xs)))
+                logging.debug('thread {}: insert for {}: {}'.format(thread_num, date_str, ','.join(str(x) for x in xs)))
```

Comment:
maybe better to use `%s` literal in logging?

Reply:
f-str is probably the best way in Python 3

Reply:
> f-str is probably the best way in Python 3

I've changed to f-strings